### ISSUE TITLE
[request] Add caching method

### ISSUE THREAD

**Issue by abordage** [m-a-k-o/nova-custom-table-card#2]
Hi. Thanks for the package.
Do you plan to add a caching method?

Like here: https://nova.laravel.com/docs/1.0/metrics/defining-metrics.html#caching

**Comment by m-a-k-o** [comment]
Hi. I'm happy that you are using my package.

I will check the possibilities and update the status asap.

**Comment by nasyrov** [comment]
Hi @m-a-k-o ,

I think this one could be closed, caching could be added easily like so:

```php
$users = Cache::remember('users', 60, function () {
    return DB::table('users')->get();
});

$this->data($users->map(...));
```

**Comment by m-a-k-o** [comment]
@nasyrov 
Indeed, keeping it open for implementation of nova standard cache method.

**Comment by nasyrov** [comment]
Although I've spotted one issue, if you register a custom card on dashboard and query database in __construct method, your query will be executed on every request to Nova.
